robot/Crisis-Management/tests/browser/t11_verify_details.robot

```diff
@@ -0,0 +1,25 @@
+*** Settings ***
+
+Resource        robot/Crisis-Management/resources/CrisisManagement.robot
+Library
+...             cumulusci.robotframework.PageObjects
+...             robot/Crisis-Management/resources/ContactPageObject.py
+...             robot/Crisis-Management/resources/CrisisManagement.py
+...             robot/Crisis-Management/resources/CommunityHomePageObject.py
+
+Suite Setup     Run keywords
+...             Open Test Browser
+
+Suite Teardown  Delete Records and Close Browser
+
+
+*** Test Cases ***
+Verify_community_user_detail
+    ${community_contact}=   API Get Id      Contact     Name=John Doe
+    Go to record home           ${community_contact}
+    Current page should be      Detail      Contact
+    Login to community as user
+    Click Element     title:John Doe
+    Click Element      title:My Account
+    Page Should Contain  COVID19 Division
+    Page Should Contain  Robot Hospital
```

Comment:
The assertion is to check it will be Robot Hospital or COVID19 Division , Will it be always those values ?

Reply:
@rjanjanam it should be the same for John Doe always

Reply:
Yes, Thanks Julian!

Reply:
once you include these two lines in your `click_user_info` keyword/function you can remove them from here. so all the test steps your robot file will just have from line 18 - 23

Reply:
Done